docs/changelog/changelog.rst

```diff
@@ -21,7 +21,10 @@ Highlights
 
 New Features
 ~~~~~~~~~~~~
-
+- :ref:`distance-task` can now be set to use only the xy coordinates in the 
```

Comment:
Suggested rewrite (to make default behavior more clear)
```
- :ref:`distance-task` by default now calculates the distance to the objective along the XY plane, to discourage flying straight up.

  If the full 3D distance is desired, set the ``3dDistance`` flag in the configuration block of the :ref:`distance-task`.
(`#360 <https://github.com/BYU-PCCL/holodeck/issues/360>`_)
```

Reply:
As per usual I like your rewrite more than mine 😞